EsApi4DScheduleServiceSampleApp/EsApi4DScheduleSampleApp/ConsoleApp.cs

```diff
@@ -23,6 +23,14 @@ public static Task<int> RunAsync(string[] args, RunAppAsync runAsync)
             {
                 IsRequired = true,
             };
+            var paginationOption = new Option<string>("--pagination", "Page size to use for pagination - must be between 1-10000.")
```

Comment:
the pagination parameter is used as a string within the query, so it's simpler to use a string for it rather than converting to an int to then convert it back into a string